Test/RetentionTimeTest.cs

```diff
@@ -0,0 +1,49 @@
+﻿using NUnit.Framework;
+using Proteomics;
+using Proteomics.ProteolyticDigestion;
+using Proteomics.RetentionTimePrediction;
+using System;
+using System.Collections.Generic;
+
+namespace Test
+{
+    [TestFixture]
+    public sealed class RetentionTimeTest
+    {
+        private static object[,] _peptides300A;
+
+        [Test]
+        public static void SimpleReversePhaseRetentionTimeTest()
```

Comment:
The purpose of this unit test is not testing real data retention time, but test the round function?

Reply:
this test is only to show that retention time calculation is working inside MM. The tests for retention time calculation are in mzlib.